articles/cosmos-db/how-to-configure-nsp.md

```diff
@@ -48,6 +48,10 @@ In cases where you have existing Azure Cosmos DB account and looking to add secu
 
 * Then select **Associate NSP** to associate this resource with your network security perimeter to enable communication with other Azure resources in the same perimeter while restricting public access to only allow the connections you specify.
 
+> [!NOTE]
+> If you are making requests to your account using the REST API, ensure that the x-ms-date header is present and in [the correct RFC 1123 date format](https://learn.microsoft.com/en-us/rest/api/cosmos-db/common-cosmosdb-rest-request-headers)
```

Comment:
```suggestion
> If you are making requests to your account using the REST API, ensure that the x-ms-date header is present and in [the correct RFC 1123 date format](https://learn.microsoft.com/rest/api/cosmos-db/common-cosmosdb-rest-request-headers)
```
this will fix the build warning